src/components/Room/UsernameForm.tsx

```diff
@@ -25,17 +28,15 @@ function UsernameForm({ handleClose }: UsernameFormProps) {
   // If we have a username in redux slice, start it with that. else, start it with empty string.
   const [inputtedUsername, setInputtedUsername] = useState<string>(username ? username : '');
   const [usernameSubmissionDone, setUsernameSubmissionDone] = useState<boolean>(false);
-
-  const submitHandler = async (event: React.MouseEvent<HTMLButtonElement>) => {
-    event.preventDefault();
-
-    // TODO: Validation - may want to use formik/yup?
-    // Make sure the username they gave isn't empty / made of only white spaces
-    const trimmedInputtedUsername: string = inputtedUsername.trim();
-    if (trimmedInputtedUsername === '') {
-      console.log('cannot have empty username');
-      // stop now! and display error. user needs to resubmit. - Or use formik/yup
-    } else {
+  const [creationFailure, setCreationFailure] = useState(false);
```

Comment:
seems like this and the line right below it are not being used for anything - delete?